src/main/scala/com/target/data_validator/GenTestData.scala

```diff
@@ -32,15 +32,19 @@ object GenTestData {
   }
 
   def main(args: Array[String]): Unit = {
-    val spark = SparkSession.builder.appName("genTestData").getOrCreate()
+    val spark = SparkSession.builder
+      .appName("genTestData")
+      .master(args.headOption.getOrElse("local"))
```

Comment:
Intending to launch this with `spark-submit`? If so probably better to use `--master <>` instead of a main arg, especially if launching to cluster mode.

Reply:
No, this is for generating test data only, mostly for local development or config testing. I think it can stay in local mode.